dotnet-testing/LoadTestingWithJmeter/UserService/Services/UserService.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Concurrent;
+using UserAPI.Models;
+
+namespace UserAPI.Services;
+
+public class UserService : IUserService
+{
+    static readonly ConcurrentBag<User> users = [
+        new User(1, "John Doe", "John.Doe@gmail.com"),
+        new User(2, "Jane Doe", "Jane.Doe@gmail.com")];
+
+    public ConcurrentBag<User> GetUsers()
+    {
+        return users;
+    }
+    public bool AddUser(User user)
```

Comment:
Just one more tiny thing, and I already made the edit in the article, but could you insert an empty line between these two methods? Thanks.